tests/CMakeLists.txt

```diff
@@ -174,40 +174,36 @@ foreach(test ${TESTS})
     WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}")
 endforeach()
 
-if(WIN32)
-    set(test_ext ".bat")
-endif()
-
 foreach(test
-  read_aes128-ctr
-  read_aes192-ctr
-  read_aes256-ctr
-  read_hmac-md5
-  read_hmac-md5-96
-  read_hmac-sha1
-  read_hmac-sha1-96
-  read_hmac-sha2-256
-  read_hmac-sha2-512
+  aes128-ctr
+  aes192-ctr
+  aes256-ctr
+  hmac-md5
+  hmac-md5-96
+  hmac-sha1
+  hmac-sha1-96
+  hmac-sha2-256
+  hmac-sha2-512
   )
   add_test(
-      NAME test_${test} COMMAND "${CMAKE_CURRENT_SOURCE_DIR}/test_${test}${test_ext}")
-  set_tests_properties(test_${test} PROPERTIES ENVIRONMENT "builddir=$<TARGET_FILE_DIR:test_read>")
+      NAME test_${test} COMMAND "${TARGET_FILE_DIR}/test_read")
+  set_tests_properties(test_${test} PROPERTIES ENVIRONMENT "FIXTURE_TEST_CRYPT=${test}")
```

Comment:
The MAC tests above need to set FIXTURE_TEST_MAC, not FIXTURE_TEST_CRYPT.

Reply:
Oh, many thanks Dan! :)